### PR TITLE
Added Traduc.com and Redacteur.com

### PR DESCRIPTION
**Pull request by Outfoxeed**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -10310,6 +10310,17 @@
             "recargapay.workable.com"
         ]
     },
+    
+    {
```
Comment (tupaschoal) [inline _data/sites.json:10314]:
```suggestion

    {
```

File: _data/sites.json [hunk 2]
```diff
@@ -10310,6 +10310,17 @@
...
+        "name": "Redacteur.com",
+        "url": "https://www.redacteur.com/identities/edit",
+        "difficulty": "easy",
+        "notes": "Login to your account. Click on \"My Account\" then \"My connexion information\". Finally click \"Delete my account\"",
```
Comment (tupaschoal) [inline _data/sites.json:10318]:
```suggestion
        "notes": "Login to your account. Click on \"My Account\" then \"My connection information\". Finally click \"Delete my account\"",
```
Should it be connection?

File: _data/sites.json [hunk 3]
```diff
@@ -12383,6 +12394,17 @@
             "tradingview.com"
         ]
     },
+    
+    {
```
Comment (tupaschoal) [inline _data/sites.json:12398]:
```suggestion

    {
```

File: _data/sites.json [hunk 4]
```diff
@@ -12383,6 +12394,17 @@
...
+        "name": "Traduc.com",
+        "url": "https://traduc.com/users/edit",
+        "difficulty": "easy",
+        "notes": "Login to your account. Click on \"My Account\" then \"My connexion information\". Finally click \"Delete my account\"",
```
Comment (tupaschoal) [inline _data/sites.json:12402]:
```suggestion
        "notes": "Login to your account. Click on \"My Account\" then \"My connection information\". Finally click \"Delete my account\"",
```

**Review by tupaschoal — COMMENTED** [review]
The two instructions are the same, are these services related in any way?

**Comment by Outfoxeed** [comment]
Yes they are made from a same company. I think they can be considered as siblings as only a few words and the look are different. Thanks for the help!

**Pull request merged**